tests/test_cli.py

```diff
@@ -0,0 +1,73 @@
+import os
+from os import path
+import pytest
+import subprocess
+from opentutor_classifier import AnswerClassifierInput
+from opentutor_classifier.svm import SVMAnswerClassifier, load_instances
+from . import fixture_path
+import re
+
+
+@pytest.fixture(autouse=True)
+def python_path_env(monkeypatch):
+    monkeypatch.setenv("PYTHONPATH", ".")
+
+
+def capture(command):
+    proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    out, err = proc.communicate()
+    return out, err, proc.returncode
+
+
+def __train_model(tmpdir) -> str:
+    test_root = tmpdir.mkdir("test")
+    training_data = fixture_path(path.join("data", "training_data.csv"))
+    model_root = os.path.join(test_root, "model_instances")
+    command = [
+        ".venv/bin/python3.8",
+        "bin/opentutor_classifier",
+        "train",
+        "--data",
+        training_data,
+        "--output",
+        model_root,
+    ]
+    out, err, exitcode = capture(command)
+    print(f"err={err}")
+    print(f"out={out}")
+    out_str = out.decode("utf-8")
+    match_out = re.findall(
+        r"M.+\/.+\n.+\=0.+[0-9]{2}\.[0-9]*\n.+\=1.+[0-9]{2}\.[0-9]*\n.+\=2.+[0-9]{2}\.[0-9]*\n",
+        out_str,
+    )
+    assert out_str == match_out[0]
+    assert len(out_str) == len(match_out[0])
+    assert exitcode == 0
+    return model_root
+
+
+def test_cli_outputs_models_at_specified_model_root(tmpdir):
```

Comment:
Here's where we should test the output. It could look something like this:

```python
def test_cli_outputs_models_at_specified_model_root(tmpdir):
    model_root, out, err, exit_code = __train_model(tmpdir)
    assert exit_code == 0
    assert path.exists(path.join(model_root, "model_instances"))
    assert path.exists(path.join(model_root, "ideal_answers"))
    assert re.search(r"Models are saved at: [^\n]+model_instances", out)
    for i in range(0, 3):
        assert re.search(
            f"Accuracy for model={i} is [0-9]+\.[0-9]+\.", out, flags=re.MULTILINE
        )
```